lib/h3/bindings/base.rb

```diff
@@ -8,7 +8,7 @@ def self.extended(base)
         base.extend FFI::Library
         base.include Structs
         base.include Types
-        base.ffi_lib ["libh3", "libh3.1"]
+        base.ffi_lib ["ext/h3/src/lib/libh3.dylib", "ext/h3/src/lib/libh3.so"]
```

Comment:
Do these relative paths work when a user uses the gem?

Reply:
Yes. Tested it locally.

Reply:
FYI I tested this badly!

After releasing `3.3.0` I added it to `stuart-api` and discovered a `LoadError`. The paths do indeed need to be absolute 🤦‍♂️ 

Hotfixed here: https://github.com/StuartApp/h3_ruby/commit/d192b41c945ba9e1c255aa8922d8067f3f84b1f6

I yanked `3.3.0` from Rubygems and released `3.3.1`. Seems to work just fine now 😅